File: 1012/1012.py
A, B, C = map(float, (input().split()))

tri = (A * C)/2
cir = 3.14159 * C * C
tra = ((A + B) / 2) * C
qua = B * B
ret = A * B

print(f'TRIANGULO: {tri:.3f}')
print(f'CIRCULO: {cir:.3f}')
print(f'TRAPEZIO: {tra:.3f}')
print(f'QUADRADO: {qua:.3f}')
print(f'RETANGULO: {ret:.3f}')

File: 1012/1012.lua
local A, B, C = io.read("*number", "*number", "*number")

local tri = (A * C)/2
local cir = 3.14159 * C * C
local tra = ((A + B) / 2) * C
local qua = B * B
local ret = A * B

print("TRIANGULO: " .. string.format("%.3f", tri))
print("CIRCULO: " .. string.format("%.3f", cir))
print("TRAPEZIO: " .. string.format("%.3f", tra))
print("QUADRADO: " .. string.format("%.3f", qua))
print("RETANGULO: " .. string.format("%.3f", ret))